warnings.md

```diff
@@ -61,7 +61,7 @@ This is _not_ a warning, but a console log message the plugin shows when it firs
 **Solution:** Change any use of `$().error(fn)` to `$().on("error", fn)`.
 
 ### \[shorthand-removed-v3\] JQMIGRATE: jQuery.fn.load() is deprecated
-### \[shorthand-removed-v3\] JQMIGRATE: jQuery.fn.unload() is deprecated
+### \[shorthand-removed-v3\] JQMIGRATE: jQuery.fn.unload() is deprecated and removed
```

Comment:
`load` should be marked as removed as well - as this particular signature did get removed: https://api.jquery.com/load-event/. What was left is https://api.jquery.com/load/; this is a different API that just shared the same name. Which one was to be used was detected by checking arguments passed to the method.

I just wonder how to do this to not confuse people about the non-event `load` method being removed... Perhaps by changing the name above to `jQuery.fn.load( [eventData ], handler )`?

Reply:
I added a commit with the suggested change to the `load()` reference. 
I agree that does make it easier to distinguish between the two loads.
Also we do mention the AJAX-related `.load()` in the **Cause:** section, so that mention along with the change to the method call hopefully is enough to highlight the difference between the two loads. 
What do you think?

Reply:
Yes, that sounds good to me. Thanks!

Reply:
Cool. Thanks.